cmd/loadTests.go

```diff
@@ -638,22 +770,36 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 				continue
 			}
 
+			/*
+				This part adds the Integration test scenario part
+				It's considered also as part of the resources creation since Integration test scenarios are resources as well
+			*/
+			var integrationTestScenario *integrationv1beta1.IntegrationTestScenario
+			integrationTestScenario, err = framework.AsKubeDeveloper.IntegrationController.CreateIntegrationTestScenario_beta1(ApplicationName, usernamespace, testScenarioGitURL, testScenarioRevision, testScenarioPathInRepo)
```

Comment:
I thought tests are defined in application/component git repository. Do we need to create them?

Reply:
Since the user needs to specify IntegrationTestScenarios in the application level, we simulate defining it ourselves.

The Integration Service is in charge of running integration test pipelines by executing
the Tekton pipeline for each **user-defined IntegrationTestScenario** (https://github.com/redhat-appstudio/book/blob/main/ADR/0015-integration-service-two-phase-architecture.md?plain=1#L12-L13)

**Users git repository path to their test code can now provide the information or the location of their IntegrationTestScenarios through Git references**
(https://github.com/redhat-appstudio/book/blob/main/ADR/0023-git-references-to-furnish-integration-test-scenarios.md?plain=1#L39-L43)
![image](https://github.com/redhat-appstudio/e2e-tests/assets/640645/21e0ce44-8d36-4279-ab3d-168b109d8194)



I guess you meant to HasController    *has.HasController (github.com/redhat-appstudio/e2e-tests/pkg/utils/has)
From which we create the CDQ and component in our e2e tests